net/raft/raft_test.go

```diff
@@ -157,6 +136,53 @@ func TestClusterSetup(t *testing.T) {
 	if got != "bar" {
 		t.Errorf("reading /foo, nodeC got %q want %q", got, "bar")
 	}
+	nodeA.cleanup()
+	nodeB.cleanup()
+	nodeC.cleanup()
+}
+
+func TestNodeEviction(t *testing.T) {
+	ctx := context.Background()
+
+	// Create new test cluster
+	nodeA, nodeB, nodeC := newTestCluster(ctx, t)
+	addrB := nodeB.addr
+
+	// Have nodeA evict nodeB
+	nodeA.service.Evict(ctx, addrB)
+	must(t, nodeA.service.WaitRead(ctx))
+	peers := nodeA.service.state.Peers()
+	for _, addr := range peers {
+		if addr == addrB {
+			t.Errorf("expected nodeB to be evicted: still in peer list")
+		}
+	}
+	nodeA.cleanup()
+	nodeB.cleanup()
+	nodeC.cleanup()
+}
+
+func TestEvictMultiple(t *testing.T) {
+	ctx := context.Background()
+
+	// Create new test cluster
+	nodeA, nodeB, nodeC := newTestCluster(ctx, t)
+	addrB := nodeB.addr
+	addrC := nodeC.addr
+
+	// Have nodeC evict nodeA and nodeB
```

Comment:
But along the lines of this comment, would it be worth testing that a node other than node A can evict the other two (either here on in an additional test)?